ECLIPs/ECLIP-etcbyz.md

```diff
@@ -0,0 +1,80 @@
+### ECLIP-?: Support for ETH Byzantium EVM and Protocol Upgrades
+
+    ECLIP: undecided
+    Title: Support for ETH Byzantium EVM and Protocol Upgrades
+    Status: Draft
+    Type: Standards Track
+    Author: Isaac Ardis <isaac.a@etclabs.org>
+    Created: 2019-02-11
+
+### Abstract
+
+Add support for a subset of protocol-impacting changes introduced in the Ethereum Foundation (ETH) network via the _Byzantium_ hardfork. The proposed changes include:
+
+- Byzantium EVM opcodes and precompiled contracts, namely opcodes `REVERT` (EIP 206/140), `RETURNDATASIZE` (EIP 211), `RETURNDATACOPY` (EIP 211), and `STATICCALL` (EIP 214/116); and precompiled contracts for modular exponentiation, elliptic curve addition, scalar multiplication, and pairing (EIPs 198, 212/197, 213/196)
+- Replacing the intermediate state root field in transaction receipts with the contract return status (EIP 658).
+
+This document proposes block `X,XXX,XXX` as the upcoming block height at which to implement these changes in the network, placing the expected date of protocol hardfork date on _XXXX-XX-XX_.
```

Comment:
This needs some thought. The release process should allow for a hard-fork on the testnets (Morden, Kotti) at least 4 weeks prior to the mainnet activation. Assuming Byzantium is well tested on Ethereum Foundation network, we can just rephrase this like

> Proposes blocks:
> 
> - `X_XXX_XXX` on Ethereum Classic mainnet
> - `X_XXX_XXX` on Morden Classic testnet
> - `X_XXX_XXX` on Kotti Classic testnet

Reply:
That said,

- Ethereum Classic block `8_750_000` will be around Wed, Sept 18, 2019
- Morden Classic block `4_723_000` will be around Wed, Aug 7, 2019
- Kotti Classic block `1_039_000` will be around Wed, Aug 7, 2019

Reply:
@soc1c can you make this as a suggestion? (cmd+g in the comment box)

Reply:
Done.

Reply:
> Edit: date should not be part of the proposal as it is a moving target naturally.

Proposing:

```suggestion
```

or?

Reply:
No. I said `date`.